e2e/testcases/custom_resources_test.go

```diff
@@ -218,21 +217,46 @@ func TestCRDDeleteBeforeRemoveCustomResourceV1(t *testing.T) {
 		nt.T.Fatal(err)
 	}
 
-	rootSyncLabels, err := nomostest.MetricLabelsForRootSync(nt, rootSyncNN)
-	if err != nil {
-		nt.T.Fatal(err)
-	}
-
-	// Wait for remediator to detect the drift (managed Anvil CR was deleted)
-	// and record it as a conflict.
-	// TODO: distinguish between management conflict (spec/generation drift) and concurrent status update conflict (resource version change)
-	err = nomostest.ValidateMetrics(nt,
-		nomostest.ReconcilerErrorMetrics(nt, rootSyncLabels, firstCommitHash, metrics.ErrorSummary{
-			Conflicts: 1,
-		}))
-	if err != nil {
-		nt.T.Fatal(err)
-	}
+	// No conflicts anymore. There was one before the reconciler refactor, because
```

Comment:
perhaps pull this long comment out to a github issue as bug report with details, mark it solved, and link to it from the code comment?

Reply:
or a buganizer link.